src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -621,6 +621,9 @@ class MessageHelper {
 
   static bool PrintSmartObject(const smart_objects::SmartObject& object);
 
+  static std::string SmartObjectToString(
```

Comment:
@AByzhynar 
Where do you use it?

Reply:
@dev-gh I used it everywhere during debug. Every time during fixing of different bugs. Not only me. This function is needed by other devs also. After defect fixed I removed usage of this function as it is needed just for debug.

Reply:
@dev-gh It is needed everytime as it allows to print out messages from hmi and mobile. It helps to determine where is the problem.

Reply:
@AByzhynar 
Perhaps 'other devs' are everyone except me. How do you think I should get that sacred  knowledge for w/o any description? So please create separate 'debug_helper' space in another file and put all tools like that there with full description. MessageHelper is overwhelmed already and needs to be cleaned up eventually.

Reply:
@dev-gh Declined as there are cases when we need to convert message to string and print it to log. Description will be added.

Reply:
@dev-gh Description added in c9b0c5c4e9bc6953544a5095a851c8e87ac0b867

Reply:
@AByzhynar 
I didn't understand what is the problem to put it to another file and call it when its needed. Can you explain particular case?

Reply:
@dev-gh I don't want  to create another MessageHelper. I consider that all things related to messages should be located in one place. I have put it to MessageHelper as other similar converters are located here.

Reply:
@AByzhynar 
By your explanation above it is debug tool and its not being used at the moment. I don't see any point to collect debug tools among number of other methods. I believe nobody but couple guys will ever know that it is there. So if you want other to use it you have to make clear its purpose. Same thing I did for application in iAP-over-Bt code and same thing did Andrey Kalinich for commands.

Reply:
@dev-gh I don't know what you did for application in iAP-over-Bt code and what did Andrey Kalinich for commands.

Reply:
@AByzhynar 
Take a look

Reply:
@dev-gh You have made a lot of changes. Where I have to look exactly? Point to the commit please.